data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1011,84 +1011,127 @@ mission "Hai Reveal [B13] Freedom"
 				`	"Can you speak to other Korath for me?"`
 					goto "translation services"
-				`	"Why were there Korath Raiders in this system?"`
-					goto korath
-				`	"I have no more questions."`
+					to display
+						not "Hai Reveal [B13] Freedom: label translation services"
+						has "Hai Reveal [B13] Freedom: label korath"
+				`	"I wish you a successful future with your newfound freedom. For now, it is time for me to depart."`
 			`	"It was a pleasure meeting you," says the human leader.`
 				goto leave
 			label korath
-			`	The Korath leader begins speak, with the Hai translating. "Several generations ago, three of our raiding ships jumped into the Alcyone system. Numerous pirate vessels left Stormhold and disabled all of our ships. A large human ship boarded our flagship while us on the other two vessels listened over communicators. All we heard was screaming. The pirates used deadly gas - what you call 'nerve gas' - to kill the entire crew. The death is excruciatingly painful to Korath. Not wanting that fate, most of the raiders on the second and third Korath ships mutinied and then surrendered. We worked as slaves for the pirate gangs, but now we're here, safe, and free. We are not specialists or their descendants like the others here, but we help as we can.`
+			action
+				set "Hai Reveal [B13] Freedom: label korath"
+			`	The Korath leader speaks, with the Hai translating. "Several generations ago, three of our raiding ships jumped to the Alcyone system. Numerous pirate vessels left Stormhold and disabled our ships. A large human ship boarded our flagship while crews on the other two vessels listened over communicators. All we heard was screaming. The pirates used deadly gas - what you call 'nerve gas' - to kill the entire crew. The death is excruciatingly painful to Korath. Not wanting that fate, most of the raiders on the other Korath ships mutinied and surrendered. We worked as slaves for the pirate gangs, but now we're here, safe, and free. We are not specialists or their descendants like the others here, but we help as we can.`
 			`	"The pirates must defend this planet against a space-based lifeform that attacks in flocks. Our ships gave them a significant advantage. Fortunately, we have one more left defend against any threats." The Korath eyes the Republic operatives as he waits for the translator to catch up.`
 			`	"I must ask you keep the presence of Korath here a secret. If the Archons or other Exiles hear of this place, they may attack to retrieve us, or the Exiles may simply pillage. We will remain here in hiding unless the Exiles finally learn to settle on their planet."`
 				goto questions
 			label "translation services"
+			action
+				set "Hai Reveal [B13] Freedom: label translation services"
 			`	The Korath leader hisses and replies while the Hai translates. "No, we cannot. Few among the Korath know your language, and we must remain here in secret. Perhaps in time there will be a way, but not yet."`
```

Comment:
```suggestion
				set "Hai Reveal [B13] Freedom: label translation services"
			`	The Korath leader hisses. "No, we cannot. We must remain here in secret. Perhaps in time there will be a way, but not yet."`
```

Reply:
The text already explicitly states that some have learned the language. However, there is no indication yet that they are physically capable of making the required sounds. Showing Korath that can speak the language is a major story change.